src/data/faq_de.json

```diff
@@ -2181,18 +2181,102 @@
+                    "description": "Im Kontext der Einführung <a href='#glossary_digital_certificate'>digitaler Zertifikate</a> für COVID-19 wurde durch das eHealth Network der Europäischen Kommission und durch die EU-Länder abgestimmt, dass jedes Land die individuellen Regeln zusammenstellt, die im grenzüberschreitenden Reiseverkehr zu prüfen sind. Diese „Business Rules“ bestehen in der Regel aus zwei Kategorien: <ul><li>Ungültigkeitsregeln (invalidation rules) des Herausgeber-Landes (country of administration), die bestimmen, unter welchen Umständen ein <a href='#glossary_digital_certificate'>digitales Zertifikat</a> nicht anerkannt wird</li><li>Akzeptanz-Regeln (acceptance rules) des Ziel-Landes (country of arrival), die erfüllt sein müssen, um in das Land einreisen zu können</li></ul>"
+                }
+            ],
+            "C": [
+                {
+                    "term": "Check-in",
+                    "anchor": "check_in",
+                    "description": "Der Check-in ist ein Verfahren, bei dem eine Person sich bei einem gewissen Ort anmeldet. Um die Check-in-Funktion der CWA für die Kontaktverfolgung anzubieten, ist es notwendig, dass im Eingangsbereich oder beim Zutritt zu einer Veranstaltung ein QR-Code veröffentlicht wird, den die Nutzer der CWA einscannen, um sich digital einzuchecken und als Kontakt erfasst zu werden."
+                },
+                {
+                    "term": "Check-out",
+                    "anchor": "check_out",
+                    "description": "Der Check-out ist ein Verfahren, bei dem eine Person sich bei einem gewissen Ort abmeldet. Der Check-out in der CWA kann automatisch durch eine durch den Veranstalter/ Organisator definierte durchschnittliche Aufenthaltsdauer erfolgen oder manuell durch den Nutzer erfolgen."
+                },
+                {
+                    "term": "Code of Conduct",
+                    "anchor": "code_of_conduct",
+                    "description": "Ein Code of Conduct (Verhaltenskodex) ist eine Sammlung von Verhaltensweisen, die in unterschiedlichen Umgebungen abhängig von der Situation generell angewendet werden sollen."
+                },
+                {
+                    "term": "Community Management",
+                    "anchor": "community_management",
+                    "description": "Im Zusammenhang mit der CWA wird über verschiedene Kanäle ein sog. Community Management betrieben. Auf diesem Weg wird sichergestellt, dass offen Fragen, technische Herausforderungen oder Schwierigkeiten bei der Nutzung der App aufgenommen und zufriedenstellend für den Nutzer beantwortet werden."

```

Comment:
"offen Fragen" => "offene Fragen"

Reply:
Will be changed in the next commit, thanks!